internal/pkg/metadb/record/record.go

```diff
@@ -30,14 +30,16 @@ var _ datastore.PropertyLoadSaver = new(PropertyMap)
 // Record represents a Open Saves record in the metadata database.
 // See the Open Saves API definition for details.
 type Record struct {
-	Key          string `datastore:"-"`
-	Blob         []byte `datastore:",noindex"`
-	BlobSize     int64
-	ExternalBlob uuid.UUID `datastore:"-"`
-	Properties   PropertyMap
-	OwnerID      string
-	Tags         []string
-	OpaqueString string `datastore:",noindex"`
+	Key            string `datastore:"-"`
+	Blob           []byte `datastore:",noindex"`
+	BlobSize       int64
+	ExternalBlob   uuid.UUID `datastore:"-"`
+	NumberOfChunks int64
```

Comment:
Do we want int64 for this? I think int32 is reasonably big.

Reply:
Yes, int32 is more than enough.

Reply:
Okay I looked at the proto and it looks like it's int64.

Reply:
I've kept int64 due the need to convert it to 32/64 back an forward, but if the conversions are ok, sure its big enough